universidade-de-fortaleza-abnt.csl

```diff
@@ -0,0 +1,741 @@
+            <text macro="collection-title" prefix=". (" suffix=")."/>
+            <!-- AQUI INSERE A COLEÇÃO OU SÉRIE DA OBRA -->
+          </group>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="collection-title">
+    <text variable="collection-title"/>
+    <text variable="collection-number" prefix=" "/>
+  </macro>
+  <macro name="genre">
+    <text variable="genre"/>
+  </macro>
+  <macro name="section">
+    <choose>
+      <if match="any" variable="section issue">
+        <text variable="section"/>
+        <text variable="issue"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <group>
+      <label variable="locator" form="short"/>
+      <text variable="locator" prefix=" "/>
+    </group>
+  </macro>
+  <macro name="place">
+    <choose>
+      <if variable="publisher-place" match="any">
+        <text variable="publisher-place" suffix=", "/>
+      </if>
+      <else-if type="paper-conference" variable="publisher-place">
+        <text variable="publisher-place" suffix=". "/>
+      </else-if>
+      <else>
+        <text value="[s. l.]" font-style="italic" suffix=", "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="archive">
+    <group>
+      <text variable="archive" prefix=" "/>
+    </group>
+  </macro>
+  <!--Citacao-->
+  <!--et al. aparece a partir de 04 autores-->
+  <citation et-al-min="4" et-al-use-first="1" collapse="year" disambiguate-add-year-suffix="true">
+    <sort>
+      <!--Puxa o autor primeiro-->
+      <key macro="author"/>
+      <!--Depois o ano-->
+      <key variable="issued"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group>
+        <text suffix=", " macro="author-short"/>
+        <text macro="issued-year"/>
+        <text prefix=", " macro="citation-locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="false" et-al-min="10" et-al-use-first="1" entry-spacing="1">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <choose>
+        <!--Projeto de lei-->
+        <if type="bill">
+          <group>
+            <!--autor-->
+            <text macro="author" suffix=". "/>
+            <!--Numero da lei-->
+            <text variable="number" suffix=". "/>
+            <!--Titulo-->
+            <text macro="title" suffix=". "/>
+            <!--Historico em negrito-->
+            <text variable="references" font-weight="bold" suffix=". "/>
+            <!-- ESTE PONTO VEM ANTES DO 'EXTRA' COMO LOCAL NAS LEIS -->
+            <!--Campo 'extra' caso queira colocar alguma observacao-->
+            <text variable="note"/>
+            <!--Dados complementares "secao, pagina"-->
+            <text macro="locators" suffix=". "/>
+          </group>
+        </if>
+        <!--Mapa-->
+        <else-if type="map">
+          <group>
+            <!--autor-->
+            <text macro="author" suffix=". "/>
```

Comment:
instead of having all these suffixes, use groups with delimiters. This makes the style more robust.